README.md

```diff
@@ -5,18 +5,23 @@
 > This is my showcase of my recently works
 
 ![screenshot](./app_screenshot.png)
+![screenshot](./app_screenshot1.png)
+![screenshot](./app_screenshot2.png)
+![screenshot](./app_screenshot3.png)
+![screenshot](./app_screenshot4.png)
+![screenshot](./app_screenshot5.png)
```

Comment:
- [ ] Let's add the screenshots to the PR as well, as they appear right now as broken links. But first, read my comment below.

[OPTIONAL] I think that only one meaningful screenshot it's enough. If the visitors of your repo are interested in the project they will visit the live demo link.